src/txmempool.cpp

```diff
@@ -457,6 +457,22 @@ void CTxMemPool::AddTransactionsUpdated(unsigned int n)
     nTransactionsUpdated += n;
 }
 
+/* Inquisition cheat to get around lack of package relay. Will only work
+ * with default minrelayfee, with non-full mempools.
+ */
+static const CFeeRate EPHEMERAL_DELTA{1000};
```

Comment:
Maybe this value should be configurable? Then you could say `-ephemeraldelta=0.0000005 -minrelaytxfee=0.0000005 -blockmintxfee=0.00001`  so that you'd only mine txs with ephemeral anchors if they were CPFPed to 1s/vb.

Reply:
I was thinking miner would override as above, everyone else would just have them all as 1000sat/kvb -- that way everything gets relayed, but it only gets mined if there's a real-ish cpfp.

Reply:
(Could leave that for another PR though)

Reply:
pushed a commit which adds `ephemeraldelta` with test